.travis.yml

```diff
@@ -31,9 +31,14 @@ install:
  - sudo mkdir -p /var/local/cache/fossology
  - sudo chown $(whoami) /var/local/cache/fossology
 
+sudo: required
+
+services:
+  - docker
+
 before_script:
  - cd src/
- - composer install --dev
```

Comment:
They should. --dev is deprecated. Dev packages are automatically installed, when `composer install` is run